README.md

```diff
@@ -1,288 +1,97 @@
+If you are not familiar with linters, read [root level README](../README.md).
 
-Modify this file to match your project and remove sections that don't apply.
+## Set-up GitHub Actions
 
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
+Please do the following **steps in this order**:
 
-OPTIONAL SECTIONS:
-- FAQ
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. When you open your first pull request you should see the result of the GitHub actions checks:
 
-After you're finished please remove all the comments and instructions!
--->
+![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
+Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
 
-  <h3><b>Microverse README Template</b></h3>
+![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
 
-</div>
+## Set-up linters in your local env
 
-<!-- TABLE OF CONTENTS -->
+**Note**: The `npm` package manager is going to create a `node_modules` directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a [`.gitignore`](https://git-scm.com/docs/gitignore) file and add `node_modules` to it:
 
-# 📗 Table of Contents
-
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
```

Comment:
- [ ] Good job creating the README file 💯, it could be better to use the README template provided by microverse that can be found [here](https://github.com/microverseinc/readme-template) 🙏 
- [ ] The README file will help people who will visit  your project to have more information about it , you should also customize the above README template with information related to your project 👍